projects/plugins/jetpack/class.json-api-endpoints.php

```diff
@@ -391,6 +391,41 @@ function cast_and_filter( $data, $documentation, $return_default_values = false,
 		return $return;
 	}
 
+	// Queries attachments to see where they have been used on the site.
+	function query_attachment_id( $attachment_id ) {
+		$queries = array( wp_get_attachment_url( $attachment_id ) );
+
+		if ( wp_attachment_is_image( $attachment_id ) ) {
+			$meta = wp_get_attachment_metadata( $attachment_id );
+
+			foreach ( $meta['sizes'] as $info ) {
+				$queries[] = wp_upload_dir()['url'] . '/' . $info['file'];
+			}
+		}
+
+		$content_ids = array();
+		foreach ( $queries as $query ) {
+			$args = array(
+				's'         => $query,
+				'fields'    => 'ids',
+				'post_type' => 'any',
+			);
+
+			$content_query = new WP_Query( $args );
```

Comment:
What if we use an [OR operator](https://developer.wordpress.org/reference/classes/wp_query/#custom-field-post-meta-parameters) instead of doing a query for each file size? 🤔

Reply:
I've tried doing some research and testing this, but my understanding is that this doesn't work with the search parameter. I haven't been able to get it to work, so I feel like it might be necessary to do individual queries for each size?